lib/core/src/lib/card-view/components/card-view-textitem/card-view-textitem.component.ts

```diff
@@ -232,4 +232,12 @@ export class CardViewTextItemComponent extends BaseCardView<CardViewTextItemMode
     get showLabelForChips(): boolean {
         return this.displayLabelForChips;
     }
+
+    get showDivider(): boolean {
```

Comment:
you have this code copy pasted across various classes, why not moving to a base class?